lib/ansible/galaxy/data/community/.github/workflows/ansible-test.yml.j2

```diff
@@ -0,0 +1,188 @@
+# README FIRST
+# 1. If you don't have unit tests remove that section
```

Comment:
@gundalow I think we should be more platform agnostic. People might have Zuul, Azure Pipeline, or GitLab Pipelines for Collection testing.

Reply:
We can add other CI platforms here (GitLab, BitBucket, etc) in the future. At that point we can ask people to remove the directories they don't need

Reply:
Got it.